Tried to paint a signal cable green

Didn't think this was going to happen. Checked the server, and it'd crashed.

I was working next to a PID, and I had a signal cable coming from the far left side. I tried to paint the cable green since I was going to run another signal cable parallel.. and this happened.

```
[15:15:37] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NoClassDefFoundError: net/minecraft/client/Minecraft
	at mods.eln.item.BrushDescriptor.use(BrushDescriptor.kt:60) ~[BrushDescriptor.class:?]
	at mods.eln.sixnode.electricalcable.ElectricalCableElement.onBlockActivated(ElectricalCableElement.java:177) ~[ElectricalCableElement.class:?]
	at mods.eln.node.six.SixNode.onBlockActivated(SixNode.java:537) ~[SixNode.class:?]
	at mods.eln.node.NodeBlockEntity.onBlockActivated(NodeBlockEntity.java:223) ~[NodeBlockEntity.class:?]
	at mods.eln.node.NodeBlock.func_149727_a(NodeBlock.java:134) ~[NodeBlock.class:?]
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:376) ~[mx.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556) ~[nh.class:?]
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60) ~[jo.class:?]
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9) ~[jo.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) ~[ej.class:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165) ~[nc.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659) ~[MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334) ~[lt.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547) ~[MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [li.class:?]
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.Minecraft
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_162]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_162]
	... 16 more
Caused by: java.lang.RuntimeException: Attempted to load class bao for invalid side SERVER
	at cpw.mods.fml.common.asm.transformers.SideTransformer.transform(SideTransformer.java:50) ~[forge.jar:?]
	at net.minecraft.launchwrapper.LaunchClassLoader.runTransformers(LaunchClassLoader.java:279) ~[launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:176) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_162]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_162]
	... 16 more
```

--- a/src/main/java/mods/eln/item/BrushDescriptor.kt
+++ b/src/main/java/mods/eln/item/BrushDescriptor.kt
@@ -57,7 +57,8 @@ class BrushDescriptor(name: String): GenericItemUsingDamageDescriptor(name) {
     }
 
     fun use(stack: ItemStack, entityPlayer: EntityPlayer): Boolean {
-        val creative = Minecraft.getMinecraft().thePlayer.capabilities.isCreativeMode
+
+        val creative = entityPlayer.capabilities.isCreativeMode
         var life = stack.tagCompound.getInteger("life")
         return if (creative || life != 0) {
             if (!creative) {

